src/views/RequestQueueView.js

```diff
@@ -58,6 +58,7 @@ class RequestQueueView extends React.Component {
       holding: PropTypes.object,
       request: PropTypes.object,
     }),
+    isTlrEnabled: PropTypes.bool.isRequired,
```

Comment:
should be optional or passed constant above should be like this const titleLevelRequestsFeatureEnabled = !!this.state?.titleLevelRequestsFeatureEnabled;

Reply:
Done, thanks!